iothub/device/samples/getting started/RoutingTutorial/SimulatedDevice/Program.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+//This is the code that sends messages to the IoT Hub for testing the routing as defined
+//  in this article: https://docs.microsoft.com/azure/iot-hub/tutorial-routing
+//The scripts for creating the resources are included in the resources folder in this
+//  Visual Studio solution. 
+//
+// This program encodes the message body so it can be queried against by the Iot hub.
+//
+// If you want to read an encoded message body, you can do this:
+//   * route the message to storage,
+//   * retrieve the message from the storage account by downloading the blob,
+//   * use the ReadOneRowFromFile method at the bottom of this module to read the first row of the file.
+//   * It will read the file, decode the first row in the file, and write it out to a new file 
+//       in ASCII so you can view it.
+
+using CommandLine;
+using Microsoft.Azure.Devices.Client;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SimulatedDevice
+{
+    internal class Program
+    {
+        private static DeviceClient s_deviceClient;
```

Comment:
Can we make this a local var for Main (with a using keyword) and pass it as a to the one helper method that needs it?